changelog.md

```diff
@@ -1,4 +1,9 @@
 # Change Log
+
+## TBD
```

Comment:
`4.2.1`

Reply:
I talked to @aklish about this. Since this can wait, we can set the version once we are ready to cut a release. Maybe it will go into `4.3.0`.